_posts/ianmoore/2016-02-1-clicky-turtle.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: post
+author: touchwick
+title: "Ian's Clicky Turtle"
+---
+This is my clicky turtle:
+
+<iframe src="https://trinket.io/embed/python/cdc4b03dbc" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+This assignment wound up being the most difficult task I've had to complete for this class. This was not, perhaps surprisingly, a result of the ```clicky``` function itself, but because I tried to incorporate a ```while``` loop. Every time I tried to run the program with the loop included, my browser crashed or demanded that the script be stopped. I tried it in both Chrome and Firefox, but found myself unable to solve the issue. Ultimately, I was forced to find a different way to complete the task. My program does not have a win condition, nor does it incorporate the ```distance``` function, which I had also hoped to use. I think this is an excellent example of why I shouldn't beat my head against the wall trying to finish something. Trying to get the first version to work took most of a day. Completing the second took less than an hour. 
```

Comment:
I dig the meta-reflection.  I promise we'll learn what you need to make your vision a reality.  But in the meantime you altered the scope of what you were trying to do to fit within the skills you had.  Well done!